wgsl/index.bs

```diff
@@ -3494,6 +3494,10 @@ The channel formats used to define those texel formats are listed in the
 The last column specifies the conversion from the stored channel bits to the value used in the shader.
 This is also known as the <dfn noexport>channel transfer function</dfn>, or CTF.
 
+Note: The channel transfer function for 8unorm maps {0,...,255} to the floating point interval [0.0, 1.0].
+
+Note: The channel transfer function for 8snorm maps {-128,...,127} to the floating point interval [-1.0, 1.0].
```

Comment:
Take either this suggestion or the next two about the ranges for consistency.
```suggestion
Note: The channel transfer function for 8unorm maps {0,...,255} to the floating point interval [0,1].

Note: The channel transfer function for 8snorm maps {-128,...,127} to the floating point interval [-1,1].
```

Reply:
took the float point one.